Assets/MixedRealityToolkit/_Core/Definitions/Utilities/MixedRealityPose.cs

```diff
@@ -35,7 +35,7 @@ public SixDof(Vector3 position)
         /// Constructor.
         /// </summary>
         /// <param name="rotation"></param>
-        public SixDof(Quaternion rotation)
+        public MixedRealityPose(Quaternion rotation)
```

Comment:
Recommend 
public MixedRealityPose(Quaternion rotation) : this(Vector3.zero, rotation) { }